extensions/ui/org.eclipse.smarthome.ui.paper/package.json

```diff
@@ -4,13 +4,15 @@
   "private": true,
   "dependencies": {
     "bower": "^1.4.1",
+    "browser-sync": "~2.7.12",
     "del": "^2.0.0",
     "gulp": "^3.9.0",
     "gulp-angular-filesort": "~1.1.1",
     "gulp-concat": "^2.6.0",
     "gulp-ng-annotate": "~1.0.0",
     "gulp-rename": "^1.2.2",
-    "gulp-uglify": "^1.5.1"
+    "gulp-uglify": "^1.5.1",
+     "http-proxy-middleware": "^0.9.0"
```

Comment:
Are browser-sync and http-proxy-middleware packaged within the final jar, so that we ship these libs? Or are they only relevant at development time?